src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -9,7 +9,7 @@
  */
 public class SortCommand extends Command {
```

Comment:
I think it would be good to change this to `SortContactCommand` (see comment below for `SortMCommand`)